mlrun/runtimes/constants.py

```diff
@@ -39,14 +55,118 @@ def default():
 class RunStates(object):
     completed = "completed"
     error = "error"
+    running = "running"
+    created = "created"
+    pending = "pending"
+    unknown = "unknown"
 
     @staticmethod
     def all():
-        return [RunStates.completed, RunStates.error]
+        return [
+            RunStates.completed,
+            RunStates.error,
+            RunStates.running,
+            RunStates.created,
+            RunStates.pending,
+            RunStates.unknown,
+        ]
 
     @staticmethod
     def stable_states():
```

Comment:
wrt our past discussion - "stable" states are not the right term here in my opinion.
these are not representing stateful resources like "cluster" "tenant" or "user" - but runtime resources. Those resources spend most of their time "running" until they complete/fail.
So it's a bit funny referring to the "final/terminal" states as stable, which insinuates those resources are "unstable" during their designated runtime even though "running" is what they do 99% of the time :)

Reply:
so what is your suggestion ? terminal states ? if so, what's the opposite, non-terminal ? transient ? running ?

Reply:
saw you suggested `in_progress` instead of `in_transient_states` that seems a bit weird cause a Pod in pending state is not exactly in progress

Reply:
Convinced 👍 , changing to terminal and non-terminal